### PR TITLE
Verifiers parsing

### PR DESCRIPTION
**Pull request by dsenchishen**

### CODE REVIEW

**Comment by corvis** [comment]
@dsenchishen linter failed

File: src/ilcdlib/dto.py [hunk 1]
```diff
@@ -110,3 +110,20 @@ class ComplianceDto:
     name: str | None
     link: str | None
     issuer: Org | None
+
+
+@dataclass(kw_only=True)
+class OrgDto:
```
Comment (corvis) [inline src/ilcdlib/dto.py:116]: Why do we need an extra DTO for Org? Use `openepd.Org`

File: src/ilcdlib/dto.py [hunk 2]
```diff
@@ -110,3 +110,20 @@ class ComplianceDto:
...
+    name: str | None
+    short_name: str | None
+    web_domain: str | None
+
+
+@dataclass(kw_only=True)
+class ValidationDto(OrgDto):
+    """Basic information about a Compliance."""
+
+    validation_type: str | None
```
Comment (corvis) [inline src/ilcdlib/dto.py:129]: add `org: Org`

File: src/ilcdlib/dto.py [hunk 3]
```diff
@@ -110,3 +110,20 @@ class ComplianceDto:
...
+    """Basic information about an Org."""
+
+    uuid: str
+    name: str | None
+    short_name: str | None
+    web_domain: str | None
+
+
+@dataclass(kw_only=True)
+class ValidationDto(OrgDto):
```
Comment (corvis) [inline src/ilcdlib/dto.py:126]: Inheritance is very unnatural here. Validation IS NOT an Org

File: src/ilcdlib/epd/reader.py [hunk 4]
```diff
@@ -680,6 +682,7 @@ def to_openepd_epd(
         epd.set_ext_field("production_location", self.get_production_location())
         epd.set_ext_field("epd_publisher", publisher)
         epd.set_ext_field("epd_developer", self.get_data_entry_by(lang, base_url))
+        epd.set_ext_field("epd_verifiers", self.get_verifiers(lang))
```
Comment (corvis) [inline src/ilcdlib/epd/reader.py:685]: Rebase on dev as we put extensions under `ilcd` key now

File: src/ilcdlib/epd/reader.py [hunk 5]
```diff
@@ -620,12 +628,6 @@ def to_openepd_epd(
         program_operator = (
             program_operator_reader.to_openepd_org(lang, base_url, provider_domain) if program_operator_reader else None
         )
```
Comment (corvis) [inline src/ilcdlib/epd/reader.py:630]: Invoke `get_verifiers` and group verifications by 2 groups: `internal` and external. Internal should go under ext, the first item from external goes to `third_party_verifier`

**Review by corvis — CHANGES_REQUESTED** [review]

File: src/ilcdlib/epd/reader.py [hunk 6]
```diff
@@ -686,7 +697,7 @@ def to_openepd_epd(
             manufacturing_description=self.get_technology_description(lang),
             product_usage_description=self.get_technological_applicability(lang),
             lca_discussion=self.get_lca_discussion(lang),
-            third_party_verifier=external_verifier,
+            third_party_verifier=self.get_third_party_verifier(lang),
```
Comment (corvis) [inline src/ilcdlib/epd/reader.py:700]: This should be `Org` object, not your dto.

File: src/ilcdlib/epd/reader.py [hunk 7]
```diff
@@ -710,6 +721,7 @@ def to_openepd_epd(
         ec3_ext = Ec3EpdExtension.construct(
             epd_developer=self.get_data_entry_by(lang, base_url),
             epd_publisher=publisher,
+            epd_verifiers=self.get_verifiers(lang),
```
Comment (corvis) [inline src/ilcdlib/epd/reader.py:724]: Let's move this under ILCD extensions

**Review by corvis — CHANGES_REQUESTED** [review]

File: src/ilcdlib/extension.py [hunk 8]
```diff
@@ -30,6 +30,7 @@ class IlcdEpdExtension(OpenEpdExtension):
     dataset_version: str | None = None
     dataset_uuid: str | None = None
     production_location: str | None = None
+    epd_verifiers: list[Org] | None = None
```
Comment (corvis) [inline src/ilcdlib/extension.py:33]: in extension it should be DTO with type field. Otherwise there is no way to understand which one is of what type.

**Review by corvis — CHANGES_REQUESTED** [review]

File: src/ilcdlib/epd/reader.py [hunk 9]
```diff
@@ -401,6 +402,26 @@ def get_openepd_compliance(self, lang: LangDef, base_url: str | None = None) ->
...
         return result
 
+    def _get_all_verifiers(self, lang: LangDef) -> list[ValidationDto]:
+        """Return list of all verifiers with type."""
+        reader = self.get_validation_reader()
+        if reader is None:
+            return []
+        return reader.get_validations(lang)
+
+    def get_verifiers(self, lang: LangDef) -> list[ValidationDto]:
```
Comment (corvis) [inline src/ilcdlib/epd/reader.py:412]: `_get_all_verifiers` is not needed any more. Move logic from `_get_all_verifiers` to this method

**Comment by dsenchishen** [comment]
> @dsenchishen Squash and rebase pls

Sorry, I missed to squash commits

**Pull request merged**